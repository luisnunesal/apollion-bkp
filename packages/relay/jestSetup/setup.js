const { JSDOM } = require('jsdom');
const server = require('./testServer');

module.exports = async () => {
  const httpServer = await server.listen(5000, () => {
    console.log(`\n test server running at: \n${JSON.stringify(httpServer.address(), null, 2)}`);
  });

  Object.assign(global, { httpServer });
};
