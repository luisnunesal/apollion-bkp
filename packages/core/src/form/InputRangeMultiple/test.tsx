import React from 'react';

import { InputRangeMultiple } from '.';

import { render, screen } from '../../__tests__/setup';

describe('Default Input Range valid props', () => {
  it('Snapshot should match default', () => {
    render(<InputRangeMultiple min="0" max="100" leftValue="25" rightValue="75" />);
    expect(screen).toMatchSnapshot();
  });
});
