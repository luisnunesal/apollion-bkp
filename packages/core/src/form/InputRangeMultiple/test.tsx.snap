// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default Input Range valid props Snapshot should match default 1`] = `
<div
  class="sc-AxiKw dxqjSV"
>
  <div
    class="sc-AxiKw cWuDWG"
  >
    <div
      class="sc-fzoNJl gPEDSH"
    >
      <span
        class="sc-fzozJi cymsSs"
        color="primary"
      >
        25
      </span>
    </div>
    <div
      class="sc-fzoNJl iLoZHt"
    >
      <span
        class="sc-fzozJi cymsSs"
        color="primary"
      >
        75
      </span>
    </div>
  </div>
  <div
    class="sc-AxiKw sc-fznJRM Qnhic"
  >
    <input
      class="sc-fznWqX dddpVb"
      max="100"
      min="0"
      type="range"
      value="25"
    />
    <input
      class="sc-fznWqX dddpVb"
      max="100"
      min="0"
      type="range"
      value="75"
    />
    <div
      class="sc-AxiKw sc-fzoiQi kIJVwy"
    >
      <div
        class="sc-AxiKw sc-fzqARJ hgDpJo"
      />
      <div
        class="sc-AxiKw sc-fzqNqU blQEEa"
      />
      <div
        class="sc-AxiKw sc-fzoyTs cDQfMn"
      />
      <div
        class="sc-AxiKw sc-fzoyTs bhfgYq"
      />
    </div>
  </div>
</div>
`;
